Fabulous.XamarinForms/templates/content/blank/.template.config/template.json

```diff
@@ -120,31 +120,265 @@
       "type": "parameter",
       "datatype": "string",
       "replaces": "AndroidSDKVersion",
-      "defaultValue": "v9.0"
+      "defaultValue": "v10.0"
     },
     "TargetAndroidAPI": {
       "type": "parameter",
       "datatype": "string",
       "replaces": "TargetAndroidAPI",
-      "defaultValue": "28"
+      "defaultValue": "29"
     },
     "MinAndroidAPI": {
       "type": "parameter",
       "datatype": "int",
       "replaces": "MinAndroidAPI",
       "defaultValue": "21"
     },
-    "XamarinAndroidSdkVersion": {
+    "XamarinAndroidXActivityVersion": {
       "type": "parameter",
       "datatype": "string",
-      "replaces": "XamarinAndroidSdkVersion",
-      "defaultValue": "28.0.0.3"
+      "replaces": "XamarinAndroidXActivityVersion",
+      "defaultValue": "1.0.0"
     },
-    "XamarinAndroidArchVersion": {
+    "XamarinAndroidXAnnotationVersion": {
       "type": "parameter",
       "datatype": "string",
-      "replaces": "XamarinAndroidArchVersion",
-      "defaultValue": "1.1.1.3"
+      "replaces": "XamarinAndroidXAnnotationVersion",
+      "defaultValue": "1.1.0"
+    },
+    "XamarinAndroidXAppCompatVersion": {
+      "type": "parameter",
+      "datatype": "string",
+      "replaces": "XamarinAndroidXAppCompatVersion",
+      "defaultValue": "1.1.0"
+    },
```

Comment:
Is it possible to aggregate several packages under a single token when they share the same version?
Like `XamarinAndroidXVersion` => `1.1.0`, `XamainAndroidXArchCoreVersion` => `2.1.0`, etc.
Would be easier to maintain I think. There's a lot of packages in Android. 😄

Reply:
Unlike in Android Support libraries, each AndroidX package can have an independent version. If we forcibly group packages together, we lose flexibility in updating.

Reply:
You can see that there are `1.0.0`, `1.0.1`, `1.1.0`, `2.0.1` and `2.1.0`. An update of an individual package will not affect other packages.

Reply:
Ok